cmd2/constants.py

```diff
@@ -6,7 +6,7 @@
 
 # Used for command parsing, tab completion and word breaks. Do not change.
 QUOTES = ['"', "'"]
```

Comment:
My gut feeling is that this file will grow over time.  There are probably more constants that can be moved here.  But this is a great start.